app.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "timdex-ui",
```

Comment:
I'm not sure what `name` does here. It thankfully isn't the name of the app that gets built.

https://devcenter.heroku.com/articles/app-json-schema#name seems to suggest it is just a label, at which point I think this is fine but I'm also not sure if it's noise.

Did you have a specific intent behind adding it? (I'm definitely not opposed, just trying to understand if it does something I don't understand or not)

Reply:
Consistency was the main intent, since we use it in our other apps, but I'm not opposed to removing if you'd prefer.

Reply:
Nope all good. If it's on the other apps, that's good enough for me.